src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -2091,14 +2089,9 @@ namespace cryptonote
     return si.available;
   }
   //-----------------------------------------------------------------------------------------------
-  const std::shared_ptr<const service_nodes::quorum_uptime_proof> core::get_uptime_quorum(uint64_t height) const
-  {
-    return m_service_node_list.get_uptime_quorum(height);
-  }
-  //-----------------------------------------------------------------------------------------------
-  const std::shared_ptr<const service_nodes::quorum_checkpointing> core::get_checkpointing_quorum(uint64_t height) const
+  const std::shared_ptr<const service_nodes::testing_quorum> core::get_testing_quorum(service_nodes::quorum_type type, uint64_t height) const
```

Comment:
fwiw returning `const std::shared_ptr` doesn't really do anything (same as returning `const int`), all that this does is help prevent RVO/move semantics.

Reply:
Removed